proposed/2023/Add-dotnet-search.md

```diff
@@ -0,0 +1,98 @@
+# **Add `search` Functionality to DotNet**
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- [Nigusu](https://github.com/Nigusu-Allehu) <!-- GitHub username link -->
+- [GitHub Issue](https://github.com/NuGet/Home/issues/6060) <!-- GitHub Issue link -->
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+The specification seeks to add search functionality to dotnet. The functionality will be added to dotnet as a search command: `dotnet package search`.  
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+The `dotnet` CLI is missing a way for customers to search for packages from the command line, as is possible with `nuget.exe search`.
+This brings NuGet experiences in the `dotnet` CLI one step closer towards feature parity.
+
+While [NuGet/Home#6060](https://github.com/NuGet/Home/issues/6060) (the linked issue) also discusses the `nuget.exe list` command, the issue was created before `nuget.exe search` was added.
```

Comment:
nit: this seems like more of an explanation about the scope rather than motivation specific.